index-october-2024-sprint.html

```diff
@@ -0,0 +1,432 @@
+     <style>
+          html {
+               font-family: Helvetica, Arial, sans-serif;
+               color: #333;
+          }
+
+          a:any-link {
+               color: #1f77b4;
+          }
+
+          #page {
+               max-width: 1000px;
+               margin: 16px auto;
+               padding: 0 12px 200px 12px;
+          }
+
+          pre {
+               display: inline;
+               white-space: normal;
+               size: 95%;
+               margin: 0 6px;
+          }
+
+          input[type=checkbox] {
+               zoom: 1.8;
+          }
+
+          .progress-container {
+               width: 100%;
+               height: 30px;
+               background-color: #EEE;
+               margin: 30px 0;
+               position: sticky;
+               top: -1px;
+          }
+
+          #progress {
+               width: 0.5%;
+               height: 100%;
+               float: left;
+               background-color: #66B66C;
+          }
+
+          #progress-text {
+               text-align: right;
+               color: #fff;
+               padding: 0.35em 0.3em 0 0;
+          }
+
+          .item {
+               margin: 20px 0;
+               line-height: 1.2;
+               font-size: 120%;
+               display: flex;
+          }
+
+          .box {
+               margin-right: 12px;
+               vertical-align: top;
+          }
+
+          .text {
+               vertical-align: top;
+               flex-grow: 1;
+          }
+
+          .done {
+               text-decoration: line-through;
+               color: #999;
+          }
+
+          .footer {
+               text-align: center;
+               margin-top: 72px;
+          }
+     </style>
+</head>
+
+<body>
+     <div id="page">
+          <h1>Checklist for the October 2024 LSST DESC Sprint Week @ SLAC</h1>
+          <h3>Here are a few useful things you can do to prepare yourself for
+               the Sprint Week!</h3>
+          <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+          <div class="progress-container">
+               <div id="progress">
+                    <div id="progress-text"></div>
+               </div>
+          </div>
+
+          <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+          <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+          <!-- Meeting resources -->
+          <h2>Access meeting resources</h2>
+          <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+          <!-- agenda page-->
+          <div class="item">
+               <div class="box">
+                    <input type="checkbox">
+               </div>
+               <div class="text">
+                    Check out and bookmark the
+                    <a href="https://confluence.slac.stanford.edu/pages/viewpage.action?pageId=500663050"
```

Comment:
This and subsequent references/links to Confluence will/may be unavailable to Feb 2025 participants.  Any critical Sprint Day information should be made available on a Google Doc or other similar cloud sharing service.  This comment applies to the subsequent references/links to confluence further down the checklist.